node/Main.hs

```diff
@@ -68,8 +69,9 @@ main = withCompileInfo $ do
     args@(CLI.SimpleNodeArgs commonNodeArgs _) <- CLI.getSimpleNodeOptions
     let loggingParams = CLI.loggingParams loggerName commonNodeArgs
     let conf          = CLI.configurationOptions (CLI.commonArgs commonNodeArgs)
+    let getKey        = cfoKey conf
```

Comment:
`configurationKey`? Then again, it's used all over the place with this convention :man_shrugging: